webextensions/manifest/theme.json

```diff
@@ -64,7 +64,8 @@
                   }
                 ],
                 "firefox_android": {
-                  "version_added": false
+                  "version_added": "65",
+                  "notes": "This property is still required."
```

Comment:
I hate to be pedantic, but why "_still_ required"? Why not just "required"?

Reply:
Yes, you're right. Updated.